### PR TITLE
feat(react.generator): add breadcrumb component

### PR DESCRIPTION
**Pull request by stif59100**
ref:MANAGER-11584

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`  
| Bug fix?         | no
| New feature?     |no
| Breaking change? | no
| Tickets          |MANAGER-11584
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I added the breadcrumb in the generator

### CODE REVIEW

File: packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx [hunk 1]
```diff
@@ -4,13 +4,20 @@
  */
 
 import React, { useEffect, useState } from 'react';
-import { Link, Params, useMatches } from 'react-router-dom';
+import { Params, useMatches } from 'react-router-dom';
+import '@ovhcloud/ods-theme-blue-jeans/index.css';
```
Comment (chipp972) [inline packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx:8]: You don't have to import this in every file, it should be only in the index

File: packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx [hunk 2]
```diff
@@ -21,43 +28,35 @@ type Match = {
...
-          {index + 1 < crumbs.length ? (
-            <Link to={pathname}>{crumb}</Link>
-          ) : (
-            <span>{crumb}</span>
-          )}
-        </li>
-      ))}
-    </ul>
-  );
+  return <OsdsBreadcrumb items={crumbs} />;
```
Comment (chipp972) [inline packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx:59]:
```suggestion
  return crumbs?.length <= 1 ? <></> : <OsdsBreadcrumb items={crumbs} />;
```

We can merge the 2 lines with a ternary

File: packages/manager/core/generator/package.json [hunk 3]
```diff
@@ -13,6 +13,10 @@
     "start": "plop"
   },
   "dependencies": {
+    "@ovhcloud/ods-core": "^13.0.1",
```
Comment (aboungnaseng-ovhcloud) [inline packages/manager/core/generator/package.json:16]: delete ods dependancies

File: packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx [hunk 4]
```diff
@@ -21,43 +28,39 @@ type Match = {
...
+        console.error('Error fetching breadcrumbs:', error);
+      });
   }, [matches]);
 
   // no need to display a breadcrumb for a single item
-  if (!crumbs || crumbs.length <= 1) {
-    return <></>;
+  if (crumbs?.length <= 1) {
+    return null;
   }
```
Comment (chipp972) [inline packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx:62]: It's repetitive with the ternary in the second return : you should remove this code

Comment (aboungnaseng-ovhcloud) [reply packages/manager/core/generator/app/templates/src/components/Breadcrumb.tsx]: Is it possible to return boolean false ?

**Review by chipp972 — APPROVED** [review]

**Pull request merged**